tests/tracer/telemetry/test_telemetry.py

```diff
@@ -50,16 +48,21 @@ def test_disable(telemetry_writer):
     assert len(httpretty.latest_requests()) == 0
 
 
-def test_fork():
-    telemetry.telemetry_writer.enable()
+def test_fork(ddtrace_run_python_code_in_subprocess):
```

Comment:
This test will be updated in the following PR to ensure the correct payloads are sent when a process forks: https://github.com/DataDog/dd-trace-py/pull/3982